addons/point_of_sale/static/src/app/screens/receipt_screen/receipt/receipt.js

```diff
@@ -34,6 +34,13 @@ export class OrderReceipt extends Component {
     get shippingDate() {
         return this.receiptEnv.shippingDate;
     }
+    /**
+     * @param {number} lineId
+     * @returns {boolean} true if the line is part of a combo
+     */
+    lineIsPartOfCombo(lineId) {
+        return this.orderlines.find((l) => l.id === lineId).isPartOfCombo();
```

Comment:
using ".id" only works if the orderline goes thru the server. Check the use of ".cid".